source/exceptions.tex

```diff
@@ -800,9 +800,9 @@
 \end{codeblock}
 The call to
 \tcode{f}
-is well-formed even though, when called,
-\tcode{f}
-might throw an exception.
+is well-formed even though
+it is possible for \tcode{f} to
+throw an exception when called.
```

Comment:
This is stating something as fact that is not known to be true -- we don't know whether it's possible or `f` to throw because we haven't seen its definition. (There's a surprisingly subtle difference between "X might Y" and "it is possible for X to Y" here.)

Reply:
Great point, reverted.

Reply:
Reverted.

Reply:
We could here say something like "even though it is not known whether `f` throws any exceptions".  (Really, it should be "even though `f` is not known not to throw any exceptions", but that sounds like a double negative.)

Reply:
Thank you, let me turn that into a secondary pull request!